src/pass/plan_memory.cc

```diff
@@ -79,14 +87,16 @@ class GraphAllocator {
   }
 
   // constructor
-  explicit GraphAllocator(const IndexedGraph* idx) : idx_(idx) {
-    this->Init(dmlc::GetEnv("NNVM_EXEC_MATCH_RANGE", 16),
```

Comment:
I'm trying to sweep between [1, NNVM_EXEC_MATCH_RANGE], and select the best value